src/redux/news-detail/news-detail.types.js

```diff
@@ -0,0 +1,2 @@
+export const GET_ARTICLE = 'GET_ARTICLE';
```

Comment:
It is better to rename it from `GET_ARTICLE` and `SET_ARTICLE` to `GET_NEWS_ARTICLE` and `SET_NEWS_ARTICLE`. It will be much more explicit. And, please, put it inside the `news.types`

Reply:
done